Unhandled exception in the UI thread.

```
GAPID Version: 0.6.2:0:developer
OS: windows 10 amd64

org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.reflect.UndeclaredThrowableException)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Unknown Source)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at com.google.gapid.Main$UI.show(Main.java:109)
	at com.google.gapid.Main.main(Main.java:67)
Caused by: java.lang.reflect.UndeclaredThrowableException
	at com.sun.proxy.$Proxy7.onStateLoaded(Unknown Source)
	at com.google.gapid.models.ApiState.fireLoadedEvent(ApiState.java:114)
	at com.google.gapid.models.ModelBase.updateSuccess(ModelBase.java:98)
	at com.google.gapid.models.ModelBase$1.onUiThreadSuccess(ModelBase.java:74)
	at com.google.gapid.rpc.UiErrorCallback.onUiThread(UiErrorCallback.java:34)
	at com.google.gapid.rpc.UiErrorCallback.onUiThread(UiErrorCallback.java:25)
	at com.google.gapid.rpc.UiCallback.lambda$onFinish$47(UiCallback.java:46)
	at com.google.gapid.widgets.Widgets.ifNotDisposed(Widgets.java:118)
	at com.google.gapid.widgets.Widgets.lambda$scheduleIfNotDisposed$1(Widgets.java:146)
	at com.google.gapid.widgets.Widgets.lambda$schedule$0(Widgets.java:138)
	at org.eclipse.swt.widgets.RunnableLock.run(Unknown Source)
	... 7 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.gapid.util.Events$ListenerCollectionImpl.invoke(Events.java:167)
	... 18 more
Caused by: java.lang.IllegalArgumentException: Index out of bounds
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.widgets.Widget.error(Unknown Source)
	at org.eclipse.swt.widgets.Tree.getItem(Unknown Source)
	at com.google.gapid.widgets.LinkifiedTree.setInput(LinkifiedTree.java:150)
	at com.google.gapid.views.StateView.onStateLoaded(StateView.java:216)
	... 23 more

I clicked on the first node under a vkQueueSubmit.
```

--- a/gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java
+++ b/gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java
@@ -146,7 +146,7 @@ public abstract class LinkifiedTree<T, F> extends Composite {
 
   public void setInput(T root) {
     viewer.setInput(root);
-    if (root != null) {
+    if (root != null && viewer.getTree().getItemCount() > 0) {
       viewer.getTree().setSelection(viewer.getTree().getItem(0));
       viewer.getTree().showSelection();
     }

